src/CampaignsAdmin/CampaignList.tsx

```diff
@@ -41,11 +41,20 @@ export const CampaignList = () => {
       title: 'Delete',
       dataIndex: 'delete',
       key: 'delete',
-      render: () => {
+      render: (_: unknown, record: definitions['Campaigns']) => {
```

Comment:
we should start using `CampaignModel` instead of the definitions from supabase, since we've augmented the file with the parsed JSON for CampaignLocationInfo